pages/hackathon/[slug]/index.js

```diff
@@ -70,7 +69,7 @@ export default function Hackathon() {
                     setSubmisssions(response.data)
                 })
                 .catch((err) => {
-                    console.error(err)
+                    //Notify to user by the error 
```

Comment:
Similarly, we can add a Modal here

Reply:
This is when the hackathon details page has loaded, but the submissions couldn't be loaded. So, maybe showing the error only on submissions tab would be better :thinking: